### ISSUE TITLE
Events shouldn't automatically have all the admins of the parent collective as admins

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#6357]
Right now when you create a new event, all the admins of the collective are automatically becoming admins of the event, worst, you cannot even remove them.

That's an issue.

An event is like a project or a working group. It's basically a subgroup of the members of the collective that come together to take care of a specific work (in this case organising an event).

The point of creating those sub groups is to decentralize and making sure that not everyone is involved in every project/event/working group (not only for power dynamics reasons but also stay sane, as nobody can keep up with everything going on, I don't want to receive request for approving an expense related to an event where I'm not involved in the actual organisation of it)

So the rule should be that every event/project/working group should have at least 1 common admin with the parent collective.

A the very least, we should be able to remove some of the admins. 
But my preference would be to only add the person that creates the event as an admin, and, as a bonus, suggest people from the collective to also add as admins.

**Comment by kewitz** [comment]
Hi @xdamman, just to clarify, the inheritance of admins between Collective and its events and projects is the default behavior since a long time ago (I can trace the function that verifies if the user is the admin of a collective or its parents to at least 2020).

I recently worked on a refactor that allows people to assign users to specific events and projects without needing to give access to the collective itself. In this refactor, I also updated the Event and Project team admin page to be explicit about the inherited admins. The previous version only displayed a message box with the message: "Team members are defined in the settings of Collective Name".

From our side, we have no interest in deprecating the behavior of admin inheritance for now, I just wanted you to know about this behavior because this can pose a security risk for your collective. I suggest reviewing the admins you have on your collectives.

**Comment by xdamman** [comment]
Yes it was like that at first for the MVP of the events feature "Good enough and safe enough to try". But with more experience on the ground, I see how this is becoming an issue now.

Glad to hear that we will soon be able to have admins of an event/project independently of the main collective. Definitely a  step in the right direction.

I'd however definitely plan on eventually deprecating this forced inheritance because it's a mismatch with the very nature of what a collective should be: a (Decentralized) Teal Organization. Such forced inheritance creates a hierarchy where the admins of the collective have now "god power" over any activity (event/working group/project) of the collective. One of the points of creating subcollectives is to empower anyone in the collective to take the lead and become a "specific source" for a given activity. In a proper flat, networked organization, every sub collective should actually be a collective in its own rights (that could eventually outgrow its parent). Forced inheritance prevents this.

To avoid abuse, the creation of such a subcollective should be approved by at least two people of the collective. Otherwise it's an issue for the host (since effectively every subcollective is a collective automatically approved).

Note that such abuse is already possible today (forced inheritance doesn't prevent this, but it helps mitigate this issue by making sure that all the admins of the parent collective are always in a position of rejecting expenses).
Good news is that such feature would be also very useful for hosts. Arguably, more than one person of the host organization should also be required to approve new collectives (you can see how this is a fractal, which could scale very well in a very organic way).

**Comment by alanna** [comment]
Related to #4611
